docs/developer-docs/latest/development/data-management.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Data Management System - Strapi Developer Docs
+description: Import, export, and transfer data using the Strapi CLI
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/export-import.html
+---
+
+# Data Management System <BetaBadge />
+
+:::callout 🚧 Feature under development
+The data management system in under development. Not all use cases are covered by the initial release. You can provide feedback about desired functionality on the [Strapi feedback website](feedback.strapi.io).
+:::
+
+Occasionally you need to move data out of or into a Strapi instance. The data management system allows you to efficiently extract data from an existing instance or archive and import that data into a separate instance. Strapi provides CLI-based commands that allow you to export and import data using providers. Common use cases include:
```

Comment:
Since we're not describing the providers for now, I would remove "using provider". One day, we could dedicate specific section of the documentation on how to create your own provider.

Reply:
removed